neuromation/api/__init__.py

```diff
@@ -145,7 +146,7 @@ async def login(
     *,
     url: URL = DEFAULT_API_URL,
     path: Optional[Path] = None,
-    timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT
+    timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT,
```

Comment:
I think this changes were introduced by `make fmt`. Should I undo them?

Reply:
It looks like a glitch of **black**. If repeated `make fmt` does not change the code you can keep it.